### DISCUSSION TITLE
dcm4che.org down?

### DISCUSSION THREAD

**Discussion by nmcalinden** [dcm4che/dcm4che#1425] [General]
Run into an issue with our gradle/maven builds with pulling in the dcm4che-core dep...

Failed to read artifact descriptor for org.dcm4che:dcm4che-core:jar:5.29.1: The following artifacts could not be resolved: org.dcm4che:dcm4che-core:pom:5.29.1 (absent): Could not transfer artifact org.dcm4che:dcm4che-core:pom:5.29.1 from/to www.dcm4che.org (https://www.dcm4che.org/maven2): www.dcm4che.org: nodename nor servname provided, or not known: Unknown host www.dcm4che.org: nodename nor servname provided, or not known

Has anything changed or a different repository url required?

**Comment by gunterze** [comment]
dcm4che.org got a new host. we are still working getting it up again.

**Comment by j4care-christianhahn** [comment]
can you please try with a new maven repository under https://maven.dcm4che.org/

**Reply by pravilla-mdsol** [reply]
This repository doesn't have the jar files of `dcm4che-net` and `dcm4che-imageio` version 2.0.26. 
This particular jar version was hosted in the http host of dcm4che.org location. Is there any other alternative, than using the latest library

**Reply by j4care-christianhahn** [reply]
could you provide your dependency declaration and which url gets resolved ?

**Reply by pravilla-mdsol** [reply]
was able to fetch my dependency from repo ` https://maven.dcm4che.org/` .
Thank you
